tests/nlu/training_data/formats/test_rasa_yaml.py

```diff
@@ -280,3 +282,97 @@ def test_minimal_yaml_nlu_file(tmp_path):
     target_file = tmp_path / "test_nlu_file.yaml"
     io_utils.write_yaml(MINIMAL_VALID_EXAMPLE, target_file, True)
     assert RasaYAMLReader.is_yaml_nlu_file(target_file)
+
+
+def test_nlg_reads_text():
+    responses_yml = textwrap.dedent(
+        """
+      responses:
+        chitchat/ask_weather:
+        - text: Where do you want to check the weather?
+    """
+    )
+
+    reader = RasaYAMLReader()
+    result = reader.reads(responses_yml)
+
+    assert result.responses == {
+        "chitchat/ask_weather": [{"text": "Where do you want to check the weather?"}]
+    }
+
+
+def test_nlg_reads_any_multimedia():
+    responses_yml = textwrap.dedent(
+        """
+      responses:
+        chitchat/ask_weather:
+        - text: Where do you want to check the weather?
+          image: https://example.com/weather.jpg
+          temperature: 25°C
+    """
+    )
+
+    reader = RasaYAMLReader()
+    result = reader.reads(responses_yml)
+
+    assert result.responses == {
+        "chitchat/ask_weather": [
+            {
+                "text": "Where do you want to check the weather?",
+                "image": "https://example.com/weather.jpg",
+                "temperature": "25°C",
+            }
+        ]
+    }
+
+
+def test_nlg_fails_to_read_empty():
+    responses_yml = textwrap.dedent(
+        """
+      responses:
+    """
+    )
+
+    reader = RasaYAMLReader()
+
+    with pytest.raises(ValueError):
+        reader.reads(responses_yml)
+
+
+def test_nlg_fails_on_empty_response():
+    responses_yml = textwrap.dedent(
+        """
+      responses:
+        chitchat/ask_weather:
+    """
+    )
+
+    reader = RasaYAMLReader()
+
+    with pytest.raises(InvalidDomain):
+        reader.reads(responses_yml)
+
+
+def test_nlg_multimedia_load_dump_roundtrip():
+    responses_yml = textwrap.dedent(
+        """
+      responses:
+        chitchat/ask_weather:
```

Comment:
optional: add another response to make sure the order isn't changed when dumping